packages/headless/src/features/facets/category-facet-set/category-facet-set-actions-loader.ts

```diff
@@ -71,6 +76,15 @@ export interface CategoryFacetSetActionCreators {
   updateCategoryFacetSortCriterion(
     payload: UpdateCategoryFacetSortCriterionActionCreatorPayload
   ): PayloadAction<UpdateCategoryFacetSortCriterionActionCreatorPayload>;
+
+  /**
+   * Updates the preventAutoSelect flag of all facets.
+   * @param payload - The action creator payload.
+   * @returns A dispatchable action.
+   */
+  updateFacetAutoSelection(
```

Comment:
Not sure were to export/load this, so I loaded it in the 2 facets that handle the action